### PR TITLE
add new blogs w/img

### PR DESCRIPTION
**Pull request by ValerieThoma**
created new branch, old branch has package-lock.json conflicts

### CODE REVIEW

File: source/_data/blogs.yml [hunk 1]
```diff
@@ -1,3 +1,28 @@
+- title: "Start your end to end testing with Angular and Cypress"
+  url: https://offering.solutions/blog/articles/2018/05/27/end-to-end-testing-with-angular-cypress/
+  sourceName: Offering Solutions Software
+  sourceUrl: https://offering.solutions/blog/
+  date: May 27, 2018
+  author: Fabian Gosebrink
+  img: /img/examples/e2e-with-angular-and-cypress.jgp
```
Comment (jennifer-shehane) [inline source/_data/blogs.yml]: Did you run this locally to ensure all of the images work correctly? I notice this has a `.jgp` extension, which does not look correct. You'll need to always run it locally and ensure the images and links are working.

Comment (ValerieThoma) [reply source/_data/blogs.yml]: Good catch. Funny thing is, I _did_ and always do run it locally. And the incorrect file extension did work. 🤷🏻‍♀️
screen shot 2018-06-05 at 10 31 44 am
screen shot 2018-06-05 at 10 31 19 am

Will fix.

Comment (ValerieThoma) [reply source/_data/blogs.yml]: Closer inspection reveals that the image extension is misspelled as well. So I guess that's why it worked. D'oh.

Comment (jennifer-shehane) [reply source/_data/blogs.yml]: 🙃

**Pull request merged**